src/auto/object.rs

```diff
@@ -38,7 +38,8 @@ pub const NONE_OBJECT: Option<&Object> = None;
 pub trait AtkObjectExt: 'static {
     fn add_relationship<P: IsA<Object>>(&self, relationship: RelationType, target: &P) -> bool;
 
-    //fn connect_property_change_handler(&self, handler: /*Unimplemented*/Fn(&Object, /*Ignored*/PropertyValues)) -> u32;
```

Comment:
This was deprecated since 2.12, and minimal 2.14 now

Reply:
So I guess it's all good, no?

Reply:
Yes. just to prevent possible questions